applications/FluidDynamicsApplication/symbolic_generation/stokes/stokes_element.lyx

```diff
@@ -0,0 +1,1005 @@
+#LyX 2.2 created this file. For more info see http://www.lyx.org/

```

Comment:
what is this file? Does it need to be in the repo?

Reply:
I guess this file is good for documentation purposes. You may open it using LyX software. I saw these similar files in a couple of other elements, so thought of keeping it.

Reply:
if it is useful in the future / for documentation then I would add it to the [Documentation Repo](https://github.com/KratosMultiphysics/Documentation)
I just checked there is nothing from FluidDynamics there yet but I think it would be a good start
Do you agree @rubenzorrilla ?

Reply:
I think this is a good idea. I don't like pretty much mixing the code with the documentation. In the README.md of the application we can add hlinks to the documentation repo.

Reply:
.lyx file is no more in this PR. So resolving the conversation.